org.buhocms.BuhoCMS.yml

```diff
@@ -10,6 +10,8 @@ finish-args:
   - --device=dri
   - --share=network
   - --filesystem=home
+cleanup-commands:
+  - 'make clean'
```

Comment:
I will reiterate: THIS HAS NO EFFECT, whatsoever. It doesn't do what you think does. At that stage all the build dirs are gone.

Reply:
Okay... I thought having it inside the module is wrong. Are you so kind as to explain possible alternatives? What should I do, so that it does, what I think it should be doing? :sweat_smile:

Reply:
the problem here is that I have to guess what you want to cleanup. But see my note about properly building git. I would have had concerns, but the PR got merged before I could raise them.

Reply:
```
make: *** No rule to make target 'clean'.  Stop.
```

at least it's consistent into what I am saying.